### ISSUE TITLE
Investigate usage of slf4bukkit

### ISSUE THREAD

**Issue by TheE** [MyWarp/MyWarp#134]
[slf4bukkit](https://github.com/rjenkinsjr/slf4bukkit) hooks input from slf4bukkit directly into Bukkit's logging system.

Currently MyWarp for Bukit bundles a log4j adapter (as does Bukkit) that hooks the output directly into Minecraft's logging system, avoiding Bukkit's logger. This is by no means ideal.

Can we switch to slf4bukkit? It seems to automatically prepend the plugin's name to the output (already done by the core) and comes with some defaults that might be problematic...

**Comment by TheE** [comment]
Blocked by https://github.com/rjenkinsjr/slf4bukkit/issues/1.
